### DISCUSSION TITLE
Student Developer Pack Keeps Getting Auto-Denied

### DISCUSSION THREAD

**Discussion by Janice999** [community/community#19905] [GitHub Education]
Hi,

I have been unable to go through the process of submitting an application for the Student Developer Pack for about a month. The option to upload files from the computer as proof is not working, and it has been excruciatingly difficult to take a photo using my webcam since it's not of highest quality. My applications keep getting auto-denied with no second look. 

Please let me know if there's anything anyone can do to help.

**Comment by baysideengineer** [comment]
Hello @Janice999, in such case you can [contact](https://support.github.com/contact/education?tags=hubberfy_education) GitHub directly. I am sure together you will find a solution for that. I wish you all the best with your application.

**Reply by antloaisa** [reply]
Hi

- +10 tries and counting...
-The camera capture is automatically rejected.
-No possibility to upload file - button disabled!?!?!?!
Auto-denied in seconds!

:C

**Reply by baysideengineer** [reply]
I understand that this can be very frustrating. Maybe you can try it [this](https://github.com/github-community/community/discussions/17736#discussioncomment-2747445) way.

**Reply by antloaisa** [reply]
Hi!

I try 6 more times... i try send repeatedly captures of my profile of student in the current institution... official id... with low quality... high quality... or selfies with information!! In laptop with Firefox, Chrome, in android mobile... The higher quality camera(back camera) its disabled on the mobile in GitHub... with Firefox and Chrome.

In ADDITION to the above information(and NEXT information), I am a student with OFFICIAL current profile in Microsoft AZURE APPROVED by Microsoft and the institution where I am studying.

They contacted each other (Microsoft-My institution) several months ago and there was no problem. Passed! Approved!

NOW with GITHUB:

I try +16 times:

A: Identification id(with selfie... various in laptop, in mobile)

B: 2 Screenshots(really 5...) of my profile on the institution's official student portal and others:
1- An image with names and arrows with indications - student email, the same approved by Microsoft and my official institution, name of the institution, my name, current date 2 times (date with day, minutes, seconds of the pc system and system date of the official student portal).
2 - The same image without indications or manipulations.

C: I contacted GitHub helpdesk and attached all this information and nothing... automatic answer... "you know the way..." :S

D: They contact in SECONDS! and they DENY automatly, both on Github app on laptop(I bought a new 1080p camera to be cooler!!! but... maybe im not cool) - Chrome and Firefox) and on mobile on android(Chrome or Firefox) and on GitHub support help as well. :S

E: A week ago it was possible to upload a capture to GitHub... now Github disabled the button (only for me or general?!?!?!?!) and only the front camera is active. "If you can't take a picture of your proof of academic status, to upload an image from your computer instead, click Upload image." ...  back camera disabled... MEEECCCCC!!!!

F: Virtual Camera OBS ... FULL SCREEN, ONLINE, highest quality... all information in the Screen of school portal... current year, month, day, hour, minute, second, name of institution, name... ... Automatic reject in seconds.!!!!

G: Virtual Camera OBS ... FULL SCREEN, ONLINE, highest quality... all information in my official identification of educational institution ... current year, institution´s name, my name, official seal of the institution, current date/day/hour/minute of pc system... Automatic reject in seconds.!!!

It's not frustrating... no... it's next level. It´s a Joke... a bad Joke... or worse...

See you!

Ps: Sorry for my bad english explanation... sometime i will make better...

**Reply by github-username-number-2** [reply]
Even the errors are all messed up and there's random HTML everywhere. Following the instructions of the errors doesn't help either:

Your application cannot be reviewed until you fix the following:
Have you completed your <a href="https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile">GitHub user profile</a> with all your relevant information, such as your full name as it appears in your image and a short bio?
You are significantly more likely to be verified if you have completed your <a href="https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile">GitHub user profile</a> with your full name as it appears in your academic affiliation document plus a short bio.
Your image should be one of the following: <br> Dated school ID

I have done all of that, nothing works, and my ID doesn't have a date on it. Also, I really dislike putting more information about myself on the internet like my full name and even my location. I have to write a bio? The screenshot is TOO SMALL?? Why would you put a minimum size limit on files and then go and only allow JPEG? Wow really smart. For such a large company this whole process sucks. It feels like one of those scam websites where you are supposed to get something for free but you never do.

**Reply by i8964** [reply]
They microsoft which had bought github are data brokers. Scammers who steal students’ information, take advantage and sell them. you see, all codes has been used to train

**Comment by tusharhow** [comment]
I understand that this can be very frustrating for you. Maybe you can try it in this [](https://github.com/github-community/community/discussions/17736#discussioncomment-2747445) way.

**Reply by antloaisa** [reply]
Hi!

Resolved by the support system several days later......

See you

**Reply by dracry0** [reply]
Do you have a solution for this? I got rejected 14 times, still counting . Tried everyway, be it camera or upload method,

**Reply by hk2281** [reply]
So i was try again after half months, and they asseted for me student pack. I used my email with official university domain, and photos of my students license with translation on same picture. And in my thoughts your should has correct bio with info and photos like your documents. I hope you can do it.

**Reply by sr5434** [reply]
For me it doesn’t even give a correct reason. It says my ID doesn’t have my school name on it and yet it clearly does. Also, the form won’t let me request manual review

**Comment by hk2281** [comment]
I have same problem with pass throw verification step, i tried 17 time by the loop with different combinations of my official student license, and every time keeping  failing like @antloaisa

**Comment by karishmathakrar** [comment]
I'm having the same exact issue. I've been denied 7 times within seconds. Please help.

**Comment by AfaGaming** [comment]
same issue

**Comment by patrickenfuego** [comment]
I'm also having this problem, I'm at 4 rejections and counting. I've gone through several of these student arrangement programs through Microsoft (who owns GitHub, oddly enough), Jetbrains, etc., and I've **never** dealt with such an infuriating process. I got so frustrated that I printed my current transcript and held it up to my camera and I was still denied. It wants me to fill out my GH bio which I did exactly as requested, yet it continues to tell me I failed this criterion.

I'm also an online student, and it denies me based on geo-location claiming that the provided document should specify this. So, you want an official document which states:

- Current date
- Whether you're local or remote
- Full name
- Institution letter head

That's absolutely *insane* - I don't think my university, which is a top 50 institution in the U.S., provides any kind of documentation with all of those items together (including my **official transcript**). In short, it's a pipe dream.

I'd just get rid of the program as it seems prohibitively difficult to validate based on your process (especially since, according to the help bot, there is no longer a manual review process). I wish I could have my 3 hours back.

**Comment by i8964** [comment]
Believe it, they microsoft which had bought github is a serious data broker. Scammers who steal students’ information, take advantage and sell them. you see, all codes has been used to train.
